docs/INSTALL.md

```diff
@@ -1,44 +1,40 @@
-# How to install Cozy-stack?
+# How to install Cozy-Stack?
 
 ## Dependencies
 
--   A reverse-proxy (nginx, caddy, haproxy, etc.)
+-   A reverse-proxy (Nginx, Caddy, HAProxy, etc.)
 -   A SMTP server
 -   CouchDB 2
--   Git
```

Comment:
I'd like to keep `git` here. It can be used by the stack to install web apps from a git repository for self-hosted.

Reply:
Ok, adding it back